### ISSUE TITLE
Color contrast issue in notebooks

### ISSUE THREAD

**Issue by anthonydresser** [microsoft/azuredatastudio#7207]
Add new code and new text buttons on hover in notebooks.

![image](https://user-images.githubusercontent.com/4324725/64899599-111f6800-d641-11e9-9d7c-53c9027420a2.png)

I find this difficult to read. Does this meet contrast ratio accessibility requirements?

**Comment by chlafreniere** [comment]
@corivera can you take a look when you have a chance?

**Comment by angdesign** [comment]
Verifying that this fails contrast ratio requirements. The proper styling for dark theme buttons have a different color value that is not reflected here. The correct blue value should be:

![image](https://user-images.githubusercontent.com/29785331/64989888-6d71ca00-d883-11e9-88fd-a734800de811.png)

Achieved by:
.notebookEditor .hoverButton:active, .notebookEditor .hoverButton:hover {
    background-color: #4894fe;
}

.notebookEditor .hoverButton {
    color: #4894fe;
}

.notebookEditor .hoverButton {
    border-color: #4894fe;
}

Per Portal dark theme patterns.

Thanks,
Andrew
